agent/engine/docker_image_manager.go

```diff
@@ -272,7 +273,7 @@ func (imageManager *dockerImageManager) removeImageState(imageStateToBeRemoved *
 	for i, imageState := range imageManager.imageStates {
 		if imageState.Image.ImageID == imageStateToBeRemoved.Image.ImageID {
 			// Image State found; hence remove it
-			seelog.Infof("Removing Image State: [%s] from Image Manager", imageState.String())
+			logger.Debug("Image Manager: removing image state", imageState.Fields())
```

Comment:
this is not removing the image state from the instance, only from the agent's internal tracking object. This seems confusing to have as a public INFO message so making it debug